javaslang/src-gen/test/java/javaslang/APITest.java

```diff
@@ -228,48 +228,42 @@ public void shouldLeftReturnNotNull() {
 
     @Test
     public void shouldFutureWithSupplierReturnNotNull() {
-        final Future<?> future = Future(() -> 1);
-        future.await();
+        final Future<?> future = Future(() -> 1).await();
```

Comment:
Thanks @valery1707 :)

In case @danieldietrich thinks this is battle-ready, could you please apply it to the usages in `FutureTest` also :)?